inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/cls/ClsResourceOperator.java

```diff
@@ -113,6 +120,31 @@ private void createTopicID(SinkInfo sinkInfo) {
         }
     }
 
+    private String getTopicID(SinkInfo sinkInfo, ClsDataNodeDTO clsDataNode, ClsSinkDTO clsSinkDTO)
+            throws TencentCloudSDKException {
+        String topicID = describeTopicIDByTopicName(sinkInfo, clsDataNode);
+        if (Strings.isEmpty(topicID)) {
+            // if topic don't exist,create topic in cls
+            topicID = createTopicReturnTopicId(clsDataNode, clsSinkDTO);
+        }
+        return topicID;
+    }
+
+    private String createTopicReturnTopicId(ClsDataNodeDTO clsDataNode, ClsSinkDTO clsSinkDTO)
+            throws TencentCloudSDKException {
+        ClsClient client = getClsClient(clsDataNode);
+        CreateTopicRequest req = getCreateTopicRequest(clsDataNode, clsSinkDTO);
+        CreateTopicResponse resp = client.CreateTopic(req);
+        LOG.info("create cls topic {} success ,topicId {}", clsSinkDTO.getTopicName(), resp.getTopicId());
```

Comment:
reate cls topic success for topicName = {}, topicId = {}

Reply:
done